ModelBundle/DataFixtures/MongoDB/LoadNodeGlobalDemoData.php

```diff
@@ -0,0 +1,188 @@
+<?php
+
+namespace OpenOrchestra\ModelBundle\DataFixtures\MongoDB;
+
+use Doctrine\Common\DataFixtures\AbstractFixture;
+use Doctrine\Common\DataFixtures\OrderedFixtureInterface;
+use Doctrine\Common\Persistence\ObjectManager;
+use OpenOrchestra\ModelBundle\DataFixtures\MongoDB\DemoContent\AbstractDataGenerator;
+use OpenOrchestra\ModelBundle\DataFixtures\MongoDB\DemoContent\CommunityDataGenerator;
+use OpenOrchestra\ModelBundle\DataFixtures\MongoDB\DemoContent\ContactDataGenerator;
+use OpenOrchestra\ModelBundle\DataFixtures\MongoDB\DemoContent\HomeRootDataGenerator;
+use OpenOrchestra\ModelBundle\DataFixtures\MongoDB\DemoContent\LegalDataGenerator;
+use OpenOrchestra\ModelBundle\DataFixtures\MongoDB\DemoContent\NewsDataGenerator;
+use OpenOrchestra\ModelInterface\Model\NodeInterface;
+use OpenOrchestra\ModelInterface\DataFixtures\OrchestraFunctionalFixturesInterface;
+use OpenOrchestra\ModelBundle\DataFixtures\MongoDB\DemoContent\Error404DataGenerator;
+use OpenOrchestra\ModelBundle\DataFixtures\MongoDB\DemoContent\Error503DataGenerator;
+use OpenOrchestra\ModelBundle\Document\Node;
+use OpenOrchestra\ModelBundle\Document\Block;
+use OpenOrchestra\DisplayBundle\DisplayBlock\Strategies\TinyMCEWysiwygStrategy;
+use OpenOrchestra\DisplayBundle\DisplayBlock\Strategies\FooterStrategy;
+
+/**
+ * Class LoadNodeGlobalDemoData
+ */
+class LoadNodeGlobalDemoData extends AbstractFixture implements OrderedFixtureInterface, OrchestraFunctionalFixturesInterface
```

Comment:
generally it generates and load the content of a global node then uses other classes to generate other nodes  that are not global